### DISCUSSION TITLE
安卓设备同时打开麦克风还有相机权限的时候PAG动画会出现卡顿现象特别卡

### DISCUSSION THREAD

**Discussion by QJF68** [Tencent/libpag#2287] [Q&A] [answered]
安卓设备同时打开麦克风还有相机权限的时候PAG动画会出现卡顿现象特别卡

**Comment by kevingpqi123** [comment, marked as answer]
pag 的渲染依赖 OpenGL 渲染环境以及当时手机的资源剩余，这种情况与手机资源的消耗可能相关
另外，由于人力有效，为了提高地接效率，一句话的反馈问题，如缺少复现demo、复现场景、必现路径等信息我们是不会进行下一步分析的，我们会优先处理问题齐全的反馈
